### PR TITLE
fix: operator & validator pubkey logs in eth1-refactor

### PR DESCRIPTION
**Pull request by moshe-blox**
- log field was called "operator_pub_key" instead of "validator" in ValidatorAdded
- operator pubkeys were logged as hex instead of base64
- log wasn't acknowledging it encountered its own OperatorAdded or ValidatorAdded events

### CODE REVIEW

File: eth/eventdatahandler/handlers.go [hunk 1]
```diff
@@ -74,12 +74,10 @@ func (edh *EventDataHandler) handleOperatorAdded(txn basedb.Txn, event *contract
 
 	if bytes.Equal(event.PublicKey, edh.operatorData.PublicKey) {
 		edh.operatorData = od
+		logger = logger.With(zap.Bool("own_operator", true))
 	}
 
 	edh.metrics.OperatorPublicKey(od.ID, od.PublicKey)
-	logger.Debug("report operator public key",
```
Comment (lior-blox) [inline eth/eventdatahandler/handlers.go:80]: @moshe-blox we dont want to show all the logs in debug?

File: eth/eventdatahandler/handlers.go [hunk 2]
```diff
@@ -208,12 +206,12 @@ func (edh *EventDataHandler) handleValidatorAdded(txn basedb.Txn, event *contrac
 	isOperatorShare := validatorShare.BelongsToOperator(edh.operatorData.ID)
 	if isOperatorShare {
 		edh.metrics.ValidatorInactive(event.PublicKey)
-		logger.Debug("processed event")
-		return validatorShare, nil
+		ownShare = validatorShare
+		logger = logger.With(zap.Bool("own_validator", true))
```
Comment (lior-blox) [inline eth/eventdatahandler/handlers.go:210]: @moshe-blox maybe committee or cluster instead of own?

File: operator/validator/task_executor.go [hunk 3]
```diff
@@ -112,6 +104,5 @@ func (c *controller) UpdateFeeRecipient(owner, recipient common.Address) error {
 		return fmt.Errorf("update validators map: %w", err)
 	}
 
-	logger.Info("executed task")
```
Comment (moshe-blox) [inline operator/validator/task_executor.go:115]: when you see the previous log of 'executing task' and no error log after it, it means the task was executed (so 'executed task' log is not necessary)

Comment (nkryuchkov) [reply operator/validator/task_executor.go:115]: @moshe-blox then maybe better "executed task" and "failed to execute task: ..."? I'd keep "executing task" as debug though to debug a possible deadlock when executing a task

**Review by nkryuchkov — APPROVED** [review]

**Pull request merged**